catalog/resource_volume.go

```diff
@@ -42,6 +42,11 @@ func ResourceVolume() common.Resource {
 	s := common.StructToSchema(VolumeInfo{},
 		func(m map[string]*schema.Schema) map[string]*schema.Schema {
 			m["storage_location"].DiffSuppressFunc = ucDirectoryPathSlashAndEmptySuppressDiff
+			m["volume_path"] = &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
```

Comment:
Only question: if this is optional, we shouldn't set this if a user specifies it. I think it probably shouldn't be optional, right?

Reply:
yeah, that's correct - it was more kind of copy/paste - we have that code in multiple places. Corrected to be computed only